### DISCUSSION TITLE
INTERNAL SERVER ERROR while requesting swags

### DISCUSSION THREAD

**Discussion by mridungupta1984** [community/community#43406] [GitHub Education] [answered]
![image](https://user-images.githubusercontent.com/119317635/210853384-aecb3a18-4bb6-4a74-891b-856826df0a0a.png)
i submitted a request for 105 swags but the system made multiple continuous request and they shown instant approval, but application form showed as follow:
![image](https://user-images.githubusercontent.com/119317635/210853726-45f18312-2436-42db-9467-7b79a9094c02.png)
i request the support team to look into this

**Comment by mridungupta1984** [comment]
@github-staff  look into this ASAP

**Comment by mridungupta1984** [comment]
@queenofcorgis @courtneycl

**Comment by mridungupta1984** [comment]
@divais i request you to look into this request also (ticket number #1956054

**Comment by samus-aran** [comment]
Hi @mridungupta1984 👋 

Thank you for providing your ticket number. We have passed this onto the internal team for investigation. We'll provide an update when available. We thank you for being patient.

**Reply by mridungupta1984** [reply]
Thank you, sir!

**Comment by mridungupta1984** [comment]
@samus-aran i request you to look for an another issue with Imprint Engine, swag partner for Github, that for the swag requests that are applied during this "Internal Server Error" issue, are recieveing Github Cheatsheets and Github Sticker, and it is mentioned for the tshirt codes but they are not generated on the account and on the invoice recieved with the parcel of Github swags.

Following are the screenshots for reference from my account and teacher friends account
![image](https://user-images.githubusercontent.com/119317635/213841953-f348f184-5810-43c4-a29f-cf790a8ee5cd.png)
![image](https://user-images.githubusercontent.com/119317635/213842639-b6657404-4743-4869-99fb-6d54153ef21a.png)
![image](https://user-images.githubusercontent.com/119317635/213842644-02458bbe-83fc-4ff8-8ab2-caefb51c990d.png)
![image](https://user-images.githubusercontent.com/119317635/213842648-48c8b460-505c-453e-bf7b-bafca4e17fef.png)
![image](https://user-images.githubusercontent.com/119317635/213842651-36fb7ec6-a41e-4d99-8a7f-1c9ec0701e13.png)
![image](https://user-images.githubusercontent.com/119317635/213842656-29e3d4c9-0903-46ce-8981-1fe79f1084fd.png)
![image](https://user-images.githubusercontent.com/119317635/213842661-a7368f06-67c6-4fcd-9076-40c3000ea86f.png)
![image](https://user-images.githubusercontent.com/119317635/213842663-e6ae71f1-3c92-4f12-90c1-803b87b64503.png)
![image](https://user-images.githubusercontent.com/119317635/213842667-6ab98b12-1492-44e3-adb0-dda5dc87eb9c.png)
![image](https://user-images.githubusercontent.com/119317635/213842672-94211c4c-c049-4182-912b-ff9a878606a0.png)
![image](https://user-images.githubusercontent.com/119317635/213842676-3fa94108-54ab-4966-90e6-08aca69f8614.png)
![image](https://user-images.githubusercontent.com/119317635/213842679-119e60ff-acfc-4d16-bb17-5f7328937624.png)
![image](https://user-images.githubusercontent.com/119317635/213842681-0c3ff104-b01c-4089-84c0-0f1ef804a01e.png)
![image](https://user-images.githubusercontent.com/119317635/213842685-a33051c6-0bd3-47ac-ba62-be1fa411d3c3.png)
![image](https://user-images.githubusercontent.com/119317635/213842690-550d572e-3719-4963-acea-ca08e36cc3a3.png)
![image](https://user-images.githubusercontent.com/119317635/213842695-69ca52f5-4166-4025-b891-e82dae18a7d1.png)
![image](https://user-images.githubusercontent.com/119317635/213842698-193a38ce-86a8-4848-b46a-0739f88616f8.png)
![image](https://user-images.githubusercontent.com/119317635/213842705-87480f8d-a5d0-417f-ab10-fd5a83afcd71.png)

some of them were approved instantly, some of them are pending for approval and some are even shipped but none of them have the details for the coupon code, just they have the mention of the item.

**Reply by samus-aran** [reply]
Hi @mridungupta1984 

Let's first look at the initial issue you're experiencing and then further explore any other issues you are having.

Reaching out to our partners providing the GitHub Teacher swag, they have removed the multiple duplicate orders. You should be able to see one (recent) approved order on your account.

They have also stated they are currently investigating the multiple submissions you had experienced. Are you able to view the original order (that created the incorrect duplicates) and confirm that you are not receiving an Internal Server Error?

Can you clarify what you mean by coupon code you are referring to in your latest comment?

**Reply by mridungupta1984** [reply]
@samus-aran by latest comment, I mean that while requesting Github swags, we get some code for github shop so that we can reward our students (that are not generated orders that are shipped)

**Reply by mridungupta1984** [reply]
@samus-aran I just tried to make a request but I am still facing the same error
![image](https://user-images.githubusercontent.com/119317635/214410065-ef2a2669-6385-49ff-964e-4cd7212f884c.png)

**Reply by mridungupta1984** [reply]
@samus-aran 
![image](https://user-images.githubusercontent.com/119317635/214410219-8bc76378-d759-4616-b748-d83491923ec9.png)
I can check that order are now sorted a bit (like mutliple entries are cancelled, and some are approved and even shipped; but all that are shipped and approved do not have the complete products in them, like in this case
![image](https://user-images.githubusercontent.com/119317635/214410679-66be3a19-aa84-419f-ba95-e40f4c7cb904.png)
here it is approved with incomplete product details because coupons codes are unavailble so that we can reward our students

**Comment by mridungupta1984** [comment]
Thank you @samus-aran  for letting @github-staff regarding the issue at the earliest.

**Comment by mridungupta1984** [comment, marked as answer]
Issue resolved by @github-staff. {Ticket with support team #1956054}

**Comment by LoemChanndany** [comment]
Of course

**Comment by gajendrabasnet** [comment]
Hey, I am Facing the same error of Duplicate orders due to Internal Server Error; two of them even got shipped, two were canceled, and one is pending, but in all of those orders neither, did I receive any emails as I used to about approval update, shipping updates and the Coupons also used to be emailed to me or be visible in the invoice. Still, they aren't showing up in the email or the Invoice.
![image](https://user-images.githubusercontent.com/96136708/218797544-66ac20ce-5f95-42b8-a520-93119a963f00.png)

**Reply by gajendrabasnet** [reply]
Hey @github-staff please look into it {Support Ticket #2013955} 
@samus-aran
